pkg/trace/sampler/prioritysampler.go

```diff
@@ -141,6 +135,10 @@ func (s *PrioritySampler) Sample(now time.Time, trace *pb.TraceChunk, root *pb.S
 	return sampled
 }
 
+func (s *PrioritySampler) EnableRemoteRates(maxTPS float64, agentVersion string) {
```

Comment:
Is it possible to pass `remoteRates` in `NewPrioritySampler()`?
This looks bit weird. Basically, we are allowing any code to call `EnableRemoteRates()`.

Reply:
Oh you're right, in my initial refactoring idea it was not possible, but I ended up not doing this refactoring and still kept it that way without remembering why I had done it. Reverting :)